<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>328</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="basicTab">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="verticalSpacing">
        <number>9</number>
       </property>
       <item row="0" column="0">
        <widget class="QRadioButton" name="rbWindowText">
         <property name="whatsThis">
          <string>Select to use text search.</string>
         </property>
         <property name="text">
          <string>&amp;Text:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="rbHandle">
         <property name="whatsThis">
          <string>Select to use handle search.</string>
         </property>
         <property name="text">
          <string>&amp;Handle:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="rbWindowClass">
         <property name="whatsThis">
          <string>Select to use class search.</string>
         </property>
         <property name="text">
          <string>&amp;Class:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="WindowClassComboBox" name="cbWindowClass">
         <property name="whatsThis">
          <string>Search for window class name.</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="HexSpinBox" name="spnHandle">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="whatsThis">
          <string>Search for window handle (HWND).</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0" colspan="3">
           <widget class="QLineEdit" name="txtWindowText">
            <property name="whatsThis">
             <string>Search for window text. Enter simple text or regular expression.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="chCaseSensitive">
            <property name="whatsThis">
             <string>Tick to enable case sensitive search.</string>
            </property>
            <property name="text">
             <string>Case &amp;Sensitive</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="chUseRegex">
            <property name="whatsThis">
             <string>Tick to enable regular expression search.</string>
            </property>
            <property name="text">
             <string>&amp;Regular Expression</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedTab">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="criteriaWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>98</width>
            <height>33</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="booleanRelationPanel" native="true">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <layout class="QVBoxLayout" name="booleanRelationPanelLayout">
              <property name="spacing">
               <number>7</number>
              </property>
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>3</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>3</number>
              </property>
              <item>
               <widget class="QWidget" name="padding" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>20</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="booleanRelationSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="itemsPanel" native="true">
             <layout class="QVBoxLayout" name="itemsPanelLayout">
              <property name="spacing">
               <number>7</number>
              </property>
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>3</number>
              </property>
              <item>
               <spacer name="itemsSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HexSpinBox</class>
   <extends>QSpinBox</extends>
   <header>ui/custom_widgets/HexSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>WindowClassComboBox</class>
   <extends>QComboBox</extends>
   <header>ui/custom_widgets/CustomComboBoxes.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>rbWindowText</tabstop>
  <tabstop>txtWindowText</tabstop>
  <tabstop>chCaseSensitive</tabstop>
  <tabstop>chUseRegex</tabstop>
  <tabstop>rbHandle</tabstop>
  <tabstop>spnHandle</tabstop>
  <tabstop>rbWindowClass</tabstop>
  <tabstop>cbWindowClass</tabstop>
 </tabstops>
 <resources>
  <include location="../../qt_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dialogButtons</sender>
   <signal>accepted()</signal>
   <receiver>FindDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogButtons</sender>
   <signal>rejected()</signal>
   <receiver>FindDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbWindowText</sender>
   <signal>clicked()</signal>
   <receiver>txtWindowText</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbHandle</sender>
   <signal>clicked()</signal>
   <receiver>spnHandle</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbWindowClass</sender>
   <signal>clicked()</signal>
   <receiver>cbWindowClass</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
